setup-cvmfs.sh

```diff
@@ -42,15 +42,22 @@ if [ "$(uname)" == "Linux" ]; then
   fi
 elif [ "$(uname)" == "Darwin" ]; then
   # Warn about the phasing out of MacOS support for this action
-  echo "::error::The CernVM-FS GitHub Action's support for MacOS has been \
-phased out with macos-10.15."
+  echo "warning The CernVM-FS GitHub Action's support for MacOS  \
+        is still experimental."
   # Temporary fix for macOS until cvmfs 2.8 is released
   if [ -z "${CVMFS_HTTP_PROXY}" ]; then
     export CVMFS_HTTP_PROXY='DIRECT'
   fi
-  brew install --cask macfuse
+  brew tap macos-fuse-t/homebrew-cask
+  brew install fuse-t
```

Comment:
How long do these steps take, and can they be cached like we do with apt archives above?

Reply:
It's 20 seconds for setup-homebrew and 20 seconds for these steps to install fuse-t. Seems like homebrew is not as easy to cache, at least I didn't find a simple recipe. So not sure if possible.

Reply:
homebrew has this: https://docs.brew.sh/Tips-N'-Tricks
and specifically "Pre-download a file for a formula" section might be interesting for us as it mentions caching, but I need to investigate a little bit

Reply:
Alternatively, it may also be faster to just directly install the .pkg for fuse-t : https://github.com/macos-fuse-t/fuse-t/?tab=readme-ov-file#other-details

Reply:
This is indeed faster (10s). I updated the setup accordingly.

Reply:
Also this might be interesting in the future: https://github.com/macos-fuse-t/fuse-t/wiki#experimental-fuse-t-framework